api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -1015,6 +1103,74 @@ public void updateFamilyName_badRequest() {
     profileController.updateProfile(profile);
   }
 
+  @Test(expected = BadRequestException.class)
+  public void adminUpdateProfile_badRequest_nullAddress() {
+    createUser();
+    Profile profile = profileController.getMe().getBody();
+    profile.setAddress(null);
+    profileController.adminUpdateProfile(profile);
+  }
+
+  @Test(expected = BadRequestException.class)
+  public void adminUpdateProfile_badRequest_nullCountry() {
```

Comment:
☠️  It will be important, as long as we have dual entry points, to write a regression test that sets every user-settable field on the profile and make sure the same thing happens in both updateProfile() and adminUpdateProfile().

It's going to be bloated, but we've basically duplicated one of the hairiest methods we have now.